.github/workflows/publish_to_wiki.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish to Wiki
+
+# Trigger this action only if there are changes pushed to the docs/** directory under the main branch
+on:
+  push:
+    paths:
+      - markdown/wiki/** # This includes all sub folders
```

Comment:
The "trigger" comment above mentions `docs/*` (which I would prefer).  But the value here says `markdown`.  I'm assuming this is a typo?  Would you consider sticking the content under `docs/wiki` instead?